src/alembic/versions/0016_add_fund_group.py

```diff
@@ -0,0 +1,30 @@
+"""add fund group column to fund table
+
+Revision ID: 0008
+Revises: 0007
+Create Date: 2021-12-17 11:00:00.000000
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+# revision identifiers, used by Alembic.
+revision = '0016'
+down_revision = '0015'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.add_column("fund", sa.Column("fund_group", sa.String(192), nullable=True))
+    op.execute("UPDATE fund SET fund_group = 'ACTIVE' WHERE original_id in ('3333','88476','88537','8888')")
```

Comment:
Another way would be to have these values in a dictionary and have a loop over them. But it is fine anyway :).